apps/emqx_authz/src/emqx_authz_schema.erl

```diff
@@ -143,10 +139,11 @@ fields(redis_cluster) ->
 http_common_fields() ->
     [ {type,            #{type => http}}
     , {enable,          #{type => boolean(), default => true}}
-    , {url,             #{type => url()}}
```

Comment:
Seems that we can't just use `url()` type to configure authz http get/post callbacks:

```erlang
> emqx_http_lib:uri_parse("http://foo.bar/${username}").
{error,{invalid_uri,":"}}
```
Also seems that this functionality never worked, so I changed schema to `base_url` + `path` + `query`.